packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -54,4 +57,36 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
         // Add the path and icon div before the `inputField`.
         this.contentNode.insertBefore(element, this.inputField);
     }
+
+    /**
+     * Append the human-readable absolute root `path` to the dialog.
+     */
+    protected appendRootName(): void {
+        if (this.workspaceService && this.workspaceService.tryGetRoots().length > 1) {
```

Comment:
Please use the utility method to determine if a multi-root workspace is opened:

https://github.com/eclipse-theia/theia/blob/743836ed2da69e16ea2583efa6ce9a505e204eb1/packages/workspace/src/browser/workspace-service.ts#L286-L288